src/test/regress/expected/multi_alter_table_add_constraints.out

```diff
@@ -437,7 +437,6 @@ HINT:  You can issue each subcommand separately
 ALTER TABLE products ADD UNIQUE(product_no);
 ERROR:  cannot create constraint without a name on a distributed table
 ALTER TABLE products ADD PRIMARY KEY(product_no);
```

Comment:
Some points of attention for testing:
- primary keys on partitioned tables
- name collisions (auto-generated name already exists)
- weird characters in column names

Reply:
Also, it is always useful to check Postgres' regression tests for the topic and make sure nothing is broken on Citus: https://github.com/postgres/postgres/blob/master/src/test/regress/sql/constraints.sql#L255